src/pykeen/models/unimodal/conv_e.py

```diff
@@ -134,6 +133,8 @@ def __init__(
         random_seed: Optional[int] = None,
         regularizer: Optional[Regularizer] = None,
         apply_batch_normalization: bool = True,
+        entity_initializer: Hint[Initializer] = 'xavier_normal',
+        relation_initializer: Hint[Initializer] = None,
```

Comment:
is making this implicit really a good idea? I think I want to make all of them explicit.

Reply:
For reproducibility / safety against silently changing the default, for sure :+1: 

I am not sure whether the impact on usability is too large.

Reply:
okay then we will make all of them explicit. This will only be a pain in HPO, but that's a problem for future, non Karneval charlie :)

Reply:
Done in a1559baf